File: Java/Fibonacci.java
package com.lev;

import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        System.out.print("Enter n:");
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int a = 0;
        int b = 1;
        int count = 2;
        System.out.print(a+" "+b+" ");
        while(count<n){
            int temp = b;
            b = a + b;
            a = temp;
            count++;
            System.out.print(b+" ");
        }

    }
}

File: C++/Fibonacci.cpp
#include<iostream>
using namespace std;
int main(){
    int a,b,count,temp,n;
    a=0;
    b=1;
    cout<<"Enter n:";
    cin>>n;
    count=2;
    cout<<a<<" "<<b<<" ";
    while(count<n){
        temp=b;
        b=a+b;
        a=temp;
        cout<<b<<" ";
        count++;
    }
    return 0;
}
